main.cpp

```diff
@@ -383,12 +485,25 @@ struct Remote
 Remote::Remote()
 {
     std::cout << "Remote Class" << std::endl;
-    std::cout << "The Remote Brand Is: " << brand << std::endl;
+    std::cout << "The Remote Brand Is: " << brand << 
+    std::endl; 
+    std::cout << "Model Number: " << modelNum << std::endl;
 }
 
 void Remote::selectInputs()
 {
-    std::cout << "Model Number: " << modelNum << std::endl;
+   for (int i=0; i < 4; ++i)
+   { 
+   if (i == 1){
+       std::cout << "Input: Cd Player" << std::endl;
+   }
+   if (i == 2){
+       std::cout << "Input: Record Player" << std::endl;
+   }
+    if (i == 3){
+       std::cout << "Input: Radio" << std::endl;
+   }
+   }
```

Comment:
add space around `=`
curly braces go on their own line with no other code
fix this indentation
- [ ] revise